### ISSUE TITLE
Check grammar in Vignettes (simple and advance)

### ISSUE THREAD

**Issue by delafuentesaiz** [midraed/water#6]

**Comment by midraed** [comment]
Esto seria importante updatear en el corto plazo.... luego de revisar la propuesta de articulo veo que las vignettes estan llenas de errores! Yo te ayudo con esto Daniel. Quizas @ffuentesp7 ?

**Comment by delafuentesaiz** [comment]
@midraed Fede, corregimos las viñetas desde el paquete, en R? o las de LATEX del paper?

**Comment by midraed** [comment]
No, son documentos separados. Habria que ir cambiando los archivos .Rmd!

o/

2015-12-03 13:59 GMT+01:00 delafuentesaiz <notifications@github.com>:

**Comment by midraed** [comment]
Any update about this @delafuentesaiz ?
